libraries/common/src/main/java/androidx/media3/common/ColorInfo.java

```diff
@@ -258,7 +302,7 @@ public boolean isValid() {
    *
    * @see Format#toLogString(Format)
    */
-  public String toLogString() {
+  public String toColorString() {
```

Comment:
Can you keep this as toLogString? The purpose being is that it is used in debug/logging and used by parent Format#toLogString(Format). 

Please change the method to combine the bit depth info into the printing. If you want, you can have that if only the bit depth info is valid, then just print the bit depths. If none at all are valid, then print NA. If all are valid then print concatenation of the information.

Reply:
I think I've modified as requested.
In VideoSampleExporter and DefaultVideoFrameProcessor I thought I should keep the existing behaviour and only check for presence of Color metadata, therefore I've added isColorValid(). isValid() is now true if at least one between bitdepth and Color info is set, so that it will enable the logging.

Reply:
Thank you for your patience. One last comment which was to change isColorValid() to isDataSpaceValid() based on https://developer.android.com/reference/android/hardware/DataSpace

Reply:
Np, it's done.

Reply:
Thank you!